src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -867,7 +897,7 @@ namespace TerminalAppLocalTests
         VERIFY_IS_TRUE(settings._profiles.at(0)._guid.has_value());
         VERIFY_IS_TRUE(settings._profiles.at(1)._guid.has_value());
         VERIFY_ARE_EQUAL(L"Windows PowerShell", settings._profiles.at(0)._name);
-        VERIFY_ARE_EQUAL(L"cmd", settings._profiles.at(1)._name);
+        VERIFY_ARE_EQUAL(L"Command Prompt", settings._profiles.at(1)._name);

```

Comment:
Every one of these is because of a test I broke when I renamed `cmd` to `Command Prompt` :|